### PR TITLE
ginkgo: Run an etcd instance for the tests

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

**Comment by tgraf** [comment]
test-me-please

File: test/docker-compose.yml [hunk 1]
```diff
@@ -3,14 +3,16 @@ services:
   consul:
     image: "consul:0.8.3"
     hostname: "consul"
-    privileged: true
```
Comment (ianvernon) [inline test/docker-compose.yml]: Maybe we need to run etcd with `privileged: true`? :/

Comment (tgraf) [reply test/docker-compose.yml]: It works fine in my local ginkgo vm

Comment (ianvernon) [reply test/docker-compose.yml]: Docker-Compose is ran on the host on Jenkins, not in the Ginkgo VM. Are you running the unit tests within the Ginkgo VM itself?

**Comment by tgraf** [comment]
test-me-please

**Comment by tgraf** [comment]
Closing in favor of #2724

**Pull request closed without merging**